### PR TITLE
Initial version

### LINKED ISSUE #2: Using Vert.x Web GraphQL handler
How to build a GraphQL server with Vert.x Web and GraphQL Java

### PR DESCRIPTION
**Pull request by tsegismont**
Closes https://github.com/vertx-howtos/vertx-howtos.github.io/issues/2

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,319 @@
+= Implementing a GraphQL server
```
Comment (jponge) [inline README.adoc:1]: Note: when you merge you will have to update https://github.com/vertx-howtos/vertx-howtos.github.io/blob/master/_data/howtos.yml

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+----
+include::src/main/resources/tasks.graphqls[]
+----
+endif::env-github[]
+
+The schema defines:
+
+* the `Task` _type_ with 3 fields: `id`, `description` and `completed`
+* the `allTasks` _query_ that returns an array of tasks and optionally takes a parameter `uncompletedOnly` (defaults to `true`)
+* the `complete` _mutation_ that takes the task `id` as parameter and returns a _boolean_ indicating whether the operation was succesful
```
Comment (jponge) [inline README.adoc]: Typo: successful

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+<2> Deploy `GraphQLVerticle`
+
+Then you can run the application:
+
+* straight from your IDE or,
+* with Maven: `mvn compile exec:java`, or
+* with Gradle: `./gradlew run` (Linux, macOS) or `gradlew run` (Windows).
+
+=== Listing uncompleted tasks
+
```
Comment (jponge) [inline README.adoc:209]: For all query executions, I'd rather show the interaction with cURL or (better) HTTPie. As it is now we can click and see the result in the browser, but when reading the document the link between the query below and the execution may not be as obvious as if the URLs was in clear ;-)

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+theme: vertx-howtos-jekyll-theme
+url: https://vertx-howtos.github.io/HOWTO-ID
```
Comment (jponge) [inline _config.yml]: graphql-howto

**Review by jponge — CHANGES_REQUESTED** [review]
Very good howto!

**Comment by tsegismont** [comment]
>Very good howto!

I had a good teacher :bowing_man:

**Review by jponge — APPROVED** [review]
Go for it 🚀

**Pull request merged**